cfgov/unprocessed/css/forms.less

```diff
@@ -606,29 +608,46 @@
     margin-bottom: unit(30px / @base-font-size-px, em);
 }
 
-.form-l_col-1-2 {
-    .respond-to-min(1060px, {
-        .grid_column(6);
-    });
+.form-label-header + .form-l .form-label-header {
+    // Override heading-5() settings.
+    .webfont-regular();
+    text-transform: none;
+    margin-bottom: unit(10px / @base-font-size-px, em);
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/91bacedd17a3af3d6286c99ffbe9f61083151b75